src/PlayAgainButton.js

```diff
@@ -4,7 +4,8 @@ import React from 'react';
 export default function PlayAgainButton(props) {
   return (
     <div
-      className='p-1 m-2 rounded-4 lets-play themed-text'
+      className='p-1 m-2 rounded-4 lets-play themed-button 
```

Comment:
There's a hard-carriage return (cr/lf) here in the className that eslint catches. Just remove the hard wrap so className is all on one line so the minify build process doesn't break the style application.

Reply:
It looks like it's a trailing space the linter doesn't like. My impression is that hardwrapping attribute strings for readability is okay, but I don't know a lot about how the minify build process works. I'm going to go ahead and put it all on one line since I don't think the wrap is particularly necessary in the case.

Reply:
I didn't see this comment earlier and I didn't get to it before the merge, I'll go ahead and do another pull for this.